syntax = "proto2";
package message;
message StartReq {
required string local_ip = 1;
required uint32 local_port = 2;
required string remote_ip = 3;
required uint32 remote_port = 4;
}
